python/rikai/spark/utils.py

```diff
@@ -54,7 +54,20 @@ def init_spark_session(conf=None, app_name="rikai", rikai_version=None):
     builder = (
         SparkSession.builder.appName(app_name)
         .config(
-            "spark.jars.packages", "ai.eto:rikai_2.12:{}".format(rikai_version)
+            "spark.jars.packages",
+            ",".join(
+                [
+                    "ai.eto:rikai_2.12:{}".format(rikai_version),
+                ]
+            ),
+        )
+        .config(
```

Comment:
do we want to have it so that these configurations are only applied for testing GCP related functionality? Could potentially cause problems if a user is on AWS and they're using these non-standard jars?

Reply:
I dont think that the production deployment can use this function at all. For example, jars, including rikai , should be located with the other jars in the OS, deployed via `spark-defaults.conf`?

For example, the above `spark.jar.packages` might overwrite deployed values as well. 

Should we mark this as internal use only?